### DISCUSSION TITLE
Export snapshot of Webhooks and Flows

### DISCUSSION THREAD

**Discussion by aron98** [directus/directus#15855] [Feature Requests]
The exportation of the schema allows for version controlling the development of a directus platform.
It would be great if all the non-instance data could be exported too like Webhooks, flows, roles and permissions.

**Comment by azrikahar** [comment]
Although there isn't an existing CLI/feature to "snapshot" data yet (you can refer to this comment on schema vs data: https://github.com/directus/directus/pull/7939#issuecomment-916307666), you should be able to "export" by fetching them via the API, then "import" with POST requests to a new instances. Here's another comment briefly going through the general idea here: https://github.com/directus/directus/discussions/9542#discussioncomment-1596860. (note that you may also need to remove primary keys before adding them to a new instance).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
